src/dropdown.test.interactions.filterable.ts

```diff
@@ -1468,7 +1470,7 @@ it('does not allow its "toggle" event to propagate', async () => {
   const host = await fixture(
     html`<glide-core-dropdown label="Label" filterable>
       <glide-core-dropdown-option
-        label=${'x'.repeat(500)}
+        label="Label"
```

Comment:
No need for a long label in this case. Probably copy-pasta from another test.

Reply:
Good find.